x-pack/solutions/observability/plugins/apm/public/components/shared/trace_waterfall/mock/trace_otel_sample.ts

```diff
@@ -0,0 +1,2370 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const traceOtelSample = [
```

Comment:
For consistency, shouldn’t this be name traceUnprocessedOtelSample? It doesn’t have either `transaction.duration.us` or `processor.event`